website/docs/docs/collaborate/model-consumption.md

```diff
@@ -1,20 +1,20 @@
 ---
-title: "Model query history"
-sidebar_label: "Model query history"
+title: "Model consumption"
+sidebar_label: "Model consumption"
 description: "Import and auto-generate exposures from dashboards and understand how models are used in downstream tools for a richer lineage."
 image: /img/docs/collaborate/dbt-explorer/model-query-queried-models.jpg
 ---
 
-# About model query history <Lifecycle status='beta' />
+# About model consumption <Lifecycle status='beta' />
 
-The model query history tile allows you to:
+The model consumption tile allows you to:
 
 - View the query count for a model based on the data warehouse's query logs.
```

Comment:
@rpourzand do we also want to describe query count or query usage with 'consumption count' and 'consumption usage'?

```suggestion
- View the consumption count for a model based on the data warehouse's query logs.
```

Reply:
Good call! It is still a query count, just more generic since it's only consumption not builds or other things. So... "view the count of queries that have consumed a given model"